tests/test_enterprise/api/test_views.py

```diff
@@ -4604,3 +4604,144 @@ def test_read_notification_request_fail_non_admin_role(
             }
         )
         assert response.status_code == 403
+
+
+@ddt.ddt
+@mark.django_db
+class TestEnterpriseCustomerReportTypesView(BaseTestEnterpriseAPIViews):
+    """
+    Test EnterpriseCustomerReportTypesView
+    """
+
+    REPORT_TYPES_ENDPOINT = 'enterprise-report-types'
+
+    def test_get_enterprise_report_types(self):
+        """
+        Test that `EnterpriseCustomerReportTypesView` returns expected response.
+        """
+        enterprise_customer = factories.EnterpriseCustomerFactory()
+        expected_report_types = {
```

Comment:
Can we put this response at a common place where multiple test can use it, for example, types with Pearson and types without Pearson.